libs/wingc/src/files.rs

```diff
@@ -38,33 +38,73 @@ impl Error for FilesError {}
 
 #[derive(Default)]
 pub struct Files {
-	data: HashMap<Utf8PathBuf, String>,
+	data: BTreeMap<Utf8PathBuf, FileData>,
+}
+
+pub struct FileData {
+	content: String,
+	is_struct_schema: bool,
```

Comment:
Once we output a 3rd file type this might become messy. Maybe a `FileKind` enum?

Reply:
I moved away from this model, now having the jsifier hold a BTree of schemas that are used to generate files.